lib/msal-common/src/authority/Authority.ts

```diff
@@ -237,6 +237,19 @@ export class Authority {
         }
     }
 
+    /**
+     * OAuth /token endpoint host for requests
+     */
+        public get tokenEndpointHost(): string {
```

Comment:
Is it needed? Isn't `tokenEndpoint` enough?

Reply:
`tokenEndpoint` includes the path which we consider pii from what I see in MSAL.net. I wanted a way to log it without the path.

Reply:
Is the path considered Pii?

Reply:
We have discussed about PII and put some guidance here - https://learn.microsoft.com/en-us/entra/msal/dotnet/resources/handling-pii 

I don't see how the token endpoint URL can have PII. The authorization URL can have PII because it can have login_hint as query parameter, but the token endpoint cannot.